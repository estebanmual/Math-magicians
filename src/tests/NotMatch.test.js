import React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import NotMatch from '../pages/NotMatch';

it('renders correctly', () => {
  render(<NotMatch />);
  expect(screen.getByText('No match for this page')).toBeInTheDocument();
});

it('renders correctly when there are no items', () => {
  const tree = renderer.create(<NotMatch />).toJSON();
  expect(tree).toMatchSnapshot();
});
