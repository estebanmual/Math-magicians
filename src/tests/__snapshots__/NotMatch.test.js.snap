// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div>
  <h3>
    No match for this page
  </h3>
</div>
`;
